frontend/packages/console-app/src/components/network-policies/_create-network-policy.scss

```diff
@@ -45,3 +45,7 @@
 .co-create-networkpolicy__rule-header .pf-m-secondary .pf-c-dropdown__toggle-icon {
   margin-right: 0;
 }
+
+.pf-c-tree-view__list-item {
```

Comment:
The TreeView component should get a className, for instance ` <TreeView className="co-create-networkpolicy__selector-preview" ...` so that this style is scoped and doesn't risk to have side effects in other parts; (hence this rule would be rewritten `.co-create-networkpolicy__selector-preview .pf-c-tree-view__list-item`)

Reply:
(or alternatively, use a surrounding class related to networkpolicy to ensure a narrow scope is used)

Reply:
In this case, I decided to use only a class selector, as I consider that showing the bullets at the left margin is a bug that should not be visible on any Tree View. E.g. without this entry:

![image](https://user-images.githubusercontent.com/939550/135625951-efc5106c-0e4f-42e0-ae0d-e3cd0812d0ac.png)

Reply:
Anyway this issue has been already fixed in PatternFly so I'll remove this code.